src/commands/services/list.ts

```diff
@@ -246,6 +253,16 @@ export default class ServicesList extends Command {
           isVisible: {
             header: "is_visible"
           },
+          isQuality: {
+            header: "is_quality",
+            get: row =>
+              SERVICEID_EXCLUSION_LIST.indexOf(row.serviceId) === 0
```

Comment:
Just to avoid doubts 
Is this `true` when the given `serviceId` is in the first position of `SERVICEID_EXCLUSION_LIST`?

Reply:
exactly, we found it yesterday and fixed it few minutes ago. Thanks for you alert